### DISCUSSION TITLE
OSPOlogy Follow Up: Layoffs and Hiring Freezes in OSPOs

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#319] [OSPO Stories]
Discuss with the community topics related to the upcoming OSPOlogy meeting:

📅 May, 31st ➡️ Layoffs and Hiring Freezes in OSPOs

* [Conference Link](https://zoom.us/j/93102110070)
* [Slides](https://docs.google.com/presentation/d/1d6LcEb7Qy8vm2_XdYgrTn2WyrINKnOl7gwOdxK6cOAY/edit?usp=sharing) and [PDF](https://github.com/todogroup/ospology/files/11612931/OSPOlogy.S3.-.Chapter.5.pdf)

![img-ospology-may](https://github.com/todogroup/ospology/assets/43671777/3c6adac9-7d49-41d0-90bd-b5cb81deb1d9)

In this open space, people can:

* Ask questions prior to the meeting starts
* Share notes from the conversation
* Make proposals for new initiatives that can help the OSPO Ecosystem
* Add useful links and resources related to the topic
* Connect with mentors involved in OSPOs and/or Open Source Management

**Comment by oleg-nenashev** [comment]
FTR the call starts at 5PM CEST according to Zoom :)

**Comment by souravcipher** [comment]
Q. How can OSPOs focus on community health in the face of layoffs and hiring freezes?
Q. What are some open issues that need to be addressed in order better to support OSPOs during times of layoffs and hiring freezes? and what are the potential solutions or strategies being explored to address these issues?

**Comment by anajsana** [comment]
from the chat: "One challenge has been the shift of the lens on contributing open source outward to now about open source use within the organization for security, exposure/legal"

**Reply by djsudduth** [reply]
Shilla's point on the health of open source reliance is tied to my comment on the lens shift on use - same concerns

**Comment by anajsana** [comment]
from the chat: As an aspiring Open Source Program Manager interested in working in an OSPO, what are the skills that are looked out for in order to gain employment or be valuable in spite of layoffs in this field?

**Comment by anajsana** [comment]
from teh chat: I think there’s a misconception that an OSPO team is only there to build a program: policies, processes, and tools, and to not think of the OSPO providing a valuable ongoing governance function across many facets of open source use and culture in an organization.

**Reply by anajsana** [reply]
Answered on chat: . Ideally, there should be a healthy partnership between security and OSPO to be successful with policies and administration.

**Comment by anajsana** [comment]
from the chat: I’m seeing LLM investigations driving internal code collaboration and accelerating open source activity

**Comment by djsudduth** [comment]
@anajsana - feels like there should be another discussion. We seemed to run out of time to think about all the considerations.
